src/components/structures/RoomSearchView.tsx

```diff
@@ -109,12 +110,7 @@ export const RoomSearchView = forwardRef<ScrollPanel, Props>(
                                             );
                                         if (!bundledRelationship || event.getThread()) continue;
                                         const room = client.getRoom(event.getRoomId());
-                                        const thread = room.findThreadForEvent(event);
-                                        if (thread) {
-                                            event.setThread(thread);
-                                        } else {
-                                            room.createThread(event.getId(), event, [], true);
-                                        }
+                                        room.createThread(event.getId(), event, [], true);
```

Comment:
This won't do `event.setThread` if the thread already exists - are we certain something else will?